recipes/unicycler/meta.yaml

```diff
@@ -1,18 +1,18 @@
 {% set name = "unicycler" %}
-{% set version = "0.4.1" %}
+{% set version = "0.4.2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 build:
   number: 0
-  skip: True
+  skip: True # [py27]
```

Comment:
Now this will be only skipped for py27. You want to skip it for everything - so that I can build it offline. Maybe add a comment, why you skip the build and that this is build offline for the time being.